packages/components/src/VirtualizedList/RowLarge/RowLarge.component.js

```diff
@@ -32,16 +32,21 @@ function RowLarge({ className, index, key, parent, style }) {
 	});
 
 	let onRowClick;
+	let onRowDoubleClick;
 	if (parent.props.onRowClick) {
 		onRowClick = event => parent.props.onRowClick(event, rowData);
 	}
+	if (parent.props.onRowDoubleClick) {
+		onRowDoubleClick = event => parent.props.onRowDoubleClick(event, rowData);
+	}
 
 	return (
 		<div
```

Comment:
Is there a lot of work to replace it by real button?

Reply:
As seen last time, yes.